react/features/always-on-top/AlwaysOnTop.js

```diff
@@ -44,7 +49,7 @@ const toolbarButtons = {
      * The descriptor of the microphone toolbar button.
      */
     microphone: {
-        classNames: [ 'button', 'icon-microphone' ],
+        classNames: [ 'button', 'icon-microphone', 'always-on-top' ],
```

Comment:
I think it could be better to make always-on-top class for the parent and not define it for every button. This will result in fewer changes in the code.